.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
if we want to install something in the before_install section you have to set this.